autorandr.py

```diff
@@ -387,6 +411,9 @@ def from_xrandr_output(cls, xrandr_output):
                 options["crtc"] = match["crtc"]
             if match["rate"]:
                 options["rate"] = match["rate"]
+            for prop in [re.sub(r"\W+", "_", p.lower()) for p in properties]:
+                if match[prop]:
+                    options["prop-" + prop] = match[prop]
```

Comment:
One thing is bad about this approach: It breaks with the tradition of having a config file where translation into xrandr arguments is very simple. I wonder if we should do something to keep that, or at least flag this as something special? (Like prefixing another `x-`..)

That's a genuine question, please don't treat this as a change request :)